src/Microsoft.DotNet.Cli.Utils/PathUtility.cs

```diff
@@ -261,6 +261,32 @@ public static string GetPathWithDirectorySeparator(string path)
             }
         }
 
+        public static string RemoveExtraPathSeparators(string path)
+        {
+            if (string.IsNullOrEmpty(path))
+            {
+                return path;
+            }
+
+            var components = path.Split(Path.DirectorySeparatorChar);
+            var result = string.Empty;
+
+            foreach (var component in components)
+            {
+                if (!string.IsNullOrEmpty(component))
+                {
+                    result = Path.Combine(result, component);
+                }
+            }
+
+            if (path[path.Length-1] == Path.DirectorySeparatorChar)
+            {
+                result += Path.DirectorySeparatorChar;
```

Comment:
If the trailing path separator was there in the input string I was thinking we should keep it.